test/postgresql.transaction.test.js

```diff
@@ -109,6 +109,33 @@ describe('transactions', function() {
     it('should see the committed insert', expectToFindPosts(post, 1));
   });
 
+  describe('on the model', function() {
+    var p1Content = {title: 'p1', content: 'post-a'};
+    var p2Content = {title: 'p2', content: 'post-a'};
+
+    before(function(done) {
+      Post.create(p1Content, function(err, p1) {
+        Post.create(p1Content, function(err, p2) {
+          done();
+        });
+      });
+    });
+
+    it('should work when operating directly on the model', function(done) {
+      Post.beginTransaction(Transaction.READ_COMMITTED, function(err, tx) {
+        Post.updateAll({content: 'post-a'}, {content: 'post-b'}, {transaction: tx},
```

Comment:
I didn't know we can pass two filters like this. I thought it would be something like `where: {or: [{ content: 'post-a'}, {content: 'post-b'}]}`. Ignore if this achieves the same result :)